chart/kubeapps/values.schema.json

```diff
@@ -1,135 +1,159 @@
-      "title": "Dashboard configuration",
       "form": true,
       "properties": {
         "replicaCount": {
-          "type": "integer",
+          "form": true,
           "title": "Number of replicas",
-          "form": true
+          "type": "integer"
         }
-      }
+      },
+      "title": "Dashboard configuration"
     },
-    "assetsvc": {
-      "type": "object",
-      "title": "Assetsvc configuration",
+    "frontend": {
       "form": true,
       "properties": {
         "replicaCount": {
-          "type": "integer",
+          "form": true,
           "title": "Number of replicas",
-          "form": true
+          "type": "integer"
         }
-      }
+      },
+      "title": "Frontend configuration"
     },
     "ingress": {
-      "type": "object",
       "form": true,
-      "title": "Ingress configuration",
       "properties": {
+        "certManager": {
+          "description": "Add the corresponding annotations for cert-manager to auto-generate the TLS certificate",
+          "form": true,
+          "hidden": {
+            "path": "ingress/enabled",
+            "value": false
+          },
+          "title": "Use cert-manager to auto-generate the TLS certificate",
+          "type": "boolean"
+        },
         "enabled": {
-          "type": "boolean",
+          "description": "Enable the ingress resource that allows you to access the Kubeapps dashboard.",
           "form": true,
           "title": "Use a custom hostname",
-          "description": "Enable the ingress resource that allows you to access the Kubeapps dashboard."
+          "type": "boolean"
         },
         "hostname": {
-          "type": "string",
           "form": true,
-          "title": "Hostname",
           "hidden": {
-            "value": false,
-            "path": "ingress/enabled"
-          }
+            "path": "ingress/enabled",
+            "value": false
+          },
+          "title": "Hostname",
+          "type": "string"
         },
         "tls": {
-          "type": "boolean",
-          "form": true,
-          "title": "Enable TLS configuration",
-          "hidden": {
-            "value": false,
-            "path": "ingress/enabled"
-          }
-        },
-        "certManager": {
-          "type": "boolean",
           "form": true,
-          "title": "Use cert-manager to auto-generate the TLS certificate",
-          "description": "Add the corresponding annotations for cert-manager to auto-generate the TLS certificate",
           "hidden": {
-            "value": false,
-            "path": "ingress/enabled"
-          }
+            "path": "ingress/enabled",
+            "value": false
+          },
+          "title": "Enable TLS configuration",
+          "type": "boolean"
         }
-      }
+      },
+      "title": "Ingress configuration"
     },
-    "authProxy": {
-      "type": "object",
-      "title": "OIDC Proxy configuration",
+    "securityContext": {
+      "form": true,
```

Comment:
I would not add the `securityContext` to the simple form. We want to keep this form as simple as possible, only including fields that people modify very often. In this case, the securityContext is not that "popular". I am okay leaving this defintion so Helm will validate the values but removing the `form` (and `hidden`) keys.

Just in case you are not aware of what `form: true` does, this is the documentation: https://github.com/kubeapps/kubeapps/blob/master/docs/developer/basic-form-support.md